### DISCUSSION TITLE
Google Tag Manager support

### DISCUSSION THREAD

**Discussion by eddgrant** [mmistakes/so-simple-theme#401] [Q&A]
## Motivation

Hey :wave: 

I'm keen to integrate Google Tag Manager (GTM) with my site, it seems that the requirements to use GTM are:

1. Paste some code as high in the `<head>` of the page as possible.
2. Additionally, paste some code immediately after the opening `<body>` tag.

I found the [jekyll-google-tag-manager plugin](https://github.com/t-richards/jekyll-google-tag-manager) which provides the necessary magic incantation via the `{% gtm head %}` and `{% gtm body %}` liquid tags. `{% gtm head %}` would work nicely with the existing `head-custom.html` include, but I can't think of a way to get `{% gtm body %}` in as the first thing in the `body` of every page.

## Suggestion

1. I may have overlooked something obvious, could you advise if there is an existing way to use `{% gtm body %}` in as the first thing in the `body` of every page?
2. Failing that, would you be open to the idea of a `body-custom.html` include, which was included directly after the opening `body` tag?

Am loving the theme by the way, keep up the great work! :pray:

**Comment by mmistakes** [comment]
There are two includes for this purpose you can override. One for adding to the `head` and another to add something near the end of the page in the`footer` element.

- https://github.com/mmistakes/so-simple-theme/blob/master/_includes/head-custom.html
- https://github.com/mmistakes/so-simple-theme/blob/master/_includes/footer-custom.html
